### ISSUE TITLE
mido-play and other tools are not pip-installed properly; complain about missing "scripts" package

### ISSUE THREAD

**Issue by andlabs** [mido/mido#568]
**Describe the bug**
Trying to run `mido-play` from a pip installation in a Python venv results in the `scripts` module, which is where the implementation is stored in the source repo, not being found. It appears this module is not even installed by pip, necessitating both a source code download AND copying the tool out of the venv `bin/`.

**To Reproduce**
1. `python3.(ver) -m venv mido`
2. `. mido/bin/activate`
3. `pip3 install mido`
4. `mido-play`

**Expected behavior**
The tool runs.

**Actual Behavior**
```
Traceback (most recent call last):
  File "/private/tmp/mido/bin/mido-play", line 5, in <module>
    from scripts.mido_play import main
ModuleNotFoundError: No module named 'scripts'
```

**Context (please complete the following information):**
- Operating system name & version: macOS 10.12
- Python version: reproduced on both 3.9.17 and 3.11.6
- mido package version (Retrievable using `mido.version_info`) & installation method (Distribution repository, PyPI, source…): <Version('1.3.0')> installed from PyPI
- backend used (amidi, portmidi, rtmidi, PyGame…): python-rtmidi

**Additional context**
After pip installation `bin/` contains
```
Activate.ps1	activate.fish	mido-ports	pip3		python3
activate	mido-connect	mido-serve	pip3.11		python3.11
activate.csh	mido-play	pip		python
```
(on the Python 3.11 venv)

(In this case I was actively intersted in using `mido-play` since it seems to be the easiest way left to send a MIDI file to an external MIDI device on macOS without having to go through a full-fledged DAW :/ )

**Comment by rdoursenaud** [comment]
Thank you for the detailed report.
I'll look into it as soon as possible.

In the meantime, you may want to [install](https://mido.readthedocs.io/en/stable/contributing.html#users) from [source](https://github.com/mido/mido/releases/tag/1.3.0) using the following command from the root of the extracted files and inside your `venv`:
```py
python -m pip install --editable .
```
I've confirmed that `mido-play` works as intended this way.

Apologies for the inconvenience.

**Comment by rdoursenaud** [comment]
Should be fixed in the newly released [version 1.3.1](https://github.com/mido/mido/releases/tag/1.3.1).

**Comment by andlabs** [comment]
Thanks! I should say that I wasn't blocked and had already done that (though on Python 3.9 I got some other werid errors I needed to work around that I didn't on Python 3.11, so eh)
